src/enumerations/content-type.ts

```diff
@@ -0,0 +1,10 @@
+enum ContentType {
+    Css = "text/css",
+    Html = "text/html",
+    Javascript = "text/javascript",
+    Json = "application/json",
+    Pdf = "application/pdf",
+    Xml = "application/xml",
+}
+
+export default ContentType;
```

Comment:
Minor, but any thoughts on using named exports across the board for consistency? I updated them all in the original port and unless there's any technical reason not to do so, I'd prefer to keep them named.

Reply:
Good point 👍